docs/trouble_shoot/trouble_shoot.rst

```diff
@@ -0,0 +1,68 @@
+Troubleshoot
+=============
+
+cannot install certain dependencies
+----
+
+fix #1
+~~~~
+
+Make ensure that local python version >= 3.7.0
+
+
+eva_server fails
+----
+
+fix #1
+~~~~
+
+Check if a process is already running on the target port (port_number). If so, terminate the process with process_id running on the port:
+
+.. code-block:: bash
+
+    sudo lsof -i :<port_number>
+    kill -9 <process_id>
+
+fix #2
+~~~~
+
+Remove eva's running directory. To do this, open a new terminal and run
+
+.. code-block:: bash
+
+    rm -rf .eva
```

Comment:
This is out of date? I think the latest version of eva does not use `.eva`?

Reply:
Yes, we moved to `eva_data`.